forge-gui/res/cardsfolder/l/leech_bonder.txt

```diff
@@ -3,7 +3,6 @@ ManaCost:2 U
 Types:Creature Merfolk Soldier
 PT:3/3
 K:etbCounter:M1M1:2
-A:AB$ Pump | Cost$ U Q | ValidTgts$ Creature | TgtPrompt$ Select target creature to remove counters | SubAbility$ DBMove | StackDescription$ None | SpellDescription$ Move a counter from target creature onto a second target creature.
-SVar:DBMove:DB$ MoveCounter | Source$ ParentTarget | ValidTgts$ Creature | TgtPrompt$ Select target creature to add counters | TargetUnique$ True | CounterType$ Any | CounterNum$ 1
+A:AB$ MoveCounter | Cost$ U Q | ValidTgts$ Creature | TgtPrompt$ Select target creature to remove counters | ValidTgts$ Creature | TargetMin$ 2 | TargetMax$ 2 | CounterType$ Any | SpellDescription$ Move a counter from target creature onto a second target creature.
```

Comment:
maybe better TgtPrompt ?

Reply:
Hmn, doesn't look like we support two different prompts this way...change seems best we can do for now?

Besides these MoveCounter cases are there more others like that so implementing an additional param as followup might be worth it?

Reply:
i meant it in a way to say "choose creatures to move counters" like you did, not need for different TgtPrompts yet imo